### PR TITLE
Add e2e tests for vsphere-ipi-disconnected 4.11

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11.yaml [hunk 1]
```diff
@@ -78,6 +78,19 @@ tests:
     - chain: cucushift-installer-rehearse-vsphere-ipi-provision
     - ref: cucushift-pre
     workflow: cucushift-e2e
+- as: e2e-vsphere-ipi-disconnected
+  interval: 24h
+  steps:
+    cluster_profile: vsphere-discon
+    env:
+      E2E_RUN_TAGS: '@vsphere-ipi and @disconnected'
```
Comment (jhou1) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11.yaml:86]: @liangxia Looking at the build artifacts here https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/27401/rehearse-27401-periodic-ci-openshift-verification-tests-master-nightly-4.11-e2e-vsphere-ipi-disconnected/1509046010603638784/artifacts/e2e-vsphere-ipi-disconnected/cucushift-e2e/build-log.txt, it appears no test case is selectred.

Comment (liangxia) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11.yaml:86]: @jhou1 Fix in https://github.com/openshift/release/pull/27404

Comment (jhou1) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11.yaml:86]: Thanks!

**Comment by jhou1** [comment]
Are they image pull due to disconnected network?

**Comment by liangxia** [comment]
> Are they image pull due to disconnected network?

Yes. E.g
```
Failed to pull image "quay.io/openshifttest/hello-openshift@sha256:eb47fdebd0f2cc0c130228ca972f15eb2858b425a3df15f10f7bb519f60f0c96": rpc error: code = Unknown desc = pinging container registry quay.io: Get "https://quay.io/v2/": dial tcp 54.144.203.57:443: i/o timeout
```

**Pull request closed without merging**